ext/bcmath/bcmath.c

```diff
@@ -577,19 +577,24 @@ PHP_FUNCTION(bccomp)
 }
 /* }}} */
 
-/* {{{ proto bool bcscale(int scale)
+/* {{{ proto int bcscale([int scale])
    Sets default scale parameter for all bc math functions */
 PHP_FUNCTION(bcscale)
 {
-	zend_long new_scale;
+	zend_long old_scale, new_scale;
 
-	ZEND_PARSE_PARAMETERS_START(1, 1)
+	ZEND_PARSE_PARAMETERS_START(0, 1)
+		Z_PARAM_OPTIONAL
 		Z_PARAM_LONG(new_scale)
 	ZEND_PARSE_PARAMETERS_END();
 
-	BCG(bc_precision) = ((int)new_scale < 0) ? 0 : new_scale;
+	old_scale = BCG(bc_precision);
+
+	if (ZEND_NUM_ARGS() == 1) {
+		BCG(bc_precision) = ((int)new_scale < 0) ? 0 : new_scale;
+	}
 
-	RETURN_TRUE;
+	RETURN_LONG(old_scale);
```

Comment:
Given that this function always returned `true`, so the return value was meaningless, I'd say this should go into 7.2.

Reply:
@remicollet, @sgolemon Okay to merge into PHP 7.2?